lib/ash_json_api/api/router.ex

```diff
@@ -69,6 +69,9 @@ defmodule AshJsonApi.Api.Router do
               ]
               |> Enum.reject(fn {_k, v} -> is_nil(v) end)
 
+            # prefix router
+            route = (prefix || "") <> route
```

Comment:
This is incorrect, the `prefix` was added here but it should instead be added by mounting our router at that path in your router, i.e `scope "the_prefix" do`

Reply:
I don't completely understand

Reply:
`prefix` is meant to be "informative", not to actually be used as the `prefix`.

```elixir
scope "/foo" do
  ..., prefix: "/foo" # we don't use the prefix to actually put `/foo` in front of each route. It is to *inform* them that they are mounted at `/foo` (because of the scope).
end
```